tests/tools/kopiarunner/kopia_snapshotter_upgrade_test.go

```diff
@@ -1,38 +0,0 @@
-package kopiarunner
-
-import (
-	"errors"
-	"testing"
-
-	"github.com/stretchr/testify/require"
-
-	"github.com/kopia/kopia/repo/format"
-)
-
-func TestUpgradeFormatVersion(t *testing.T) {
-	repoDir := t.TempDir()
-
-	ks, err := NewKopiaSnapshotter(repoDir)
-	if errors.Is(err, ErrExeVariableNotSet) {
-		t.Skip("KOPIA_EXE not set, skipping test")
-	}
-	// Create repo in an old version
-	err = ks.CreateRepo("filesystem", "--path="+repoDir, "--format-version", "1")
-	require.NoError(t, err)
-
-	rs, err := ks.GetRepositoryStatus()
-	require.NoError(t, err)
-
-	prev := rs.ContentFormat.MutableParameters.Version
-	require.Equal(t, prev, format.Version(1), "The format version should be 1.")
-
-	ks.UpgradeRepository(repoDir)
-
-	rs, err = ks.GetRepositoryStatus()
-	require.NoError(t, err)
-
-	got := rs.ContentFormat.MutableParameters.Version
-	require.Equal(t, got, format.Version(2), "The format version should be upgraded to 2.")
-
-	require.NotEqual(t, got, prev, "The format versions should be different.")
-}
```

Comment:
Forgive my ignorance, why is this removed?

Reply:
I moved the functions GetRepositoryStatus and UpgradeRepository to snapmeta package. May be I can add a test there.

Reply:
Actually, both the functions are exercised in the robustness test, so, do we need another test here?

Reply:
Unit tests are all always a good idea, if they are applicable. At the very least we can check that `GetRepositoryStatus` does not return error on `Run()` or parse.

Reply:
Added test